config-connector/solutions/projects/kpt/project-hierarchy/folder.yaml

```diff
@@ -0,0 +1,23 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+apiVersion: resourcemanager.cnrm.cloud.google.com/v1beta1
+kind: Folder
+metadata:
+  annotations:
+    # Replace "${ORG_ID?}" with the numeric ID for your organization
```

Comment:
By seeing this, I thought I need to manually replace the placeholder with the values. Should we say "# Replace "${ORG_ID?}" with the numeric ID for your organization using `kpt`"?

Reply:
I think either way will work.

Reply:
I'm not sure we should try to railroad users into using kpt setters.

Reply:
I think we should because this is a kpt solution. We may have a different comment in other solutions.

Reply:
As Alex said, our focus is on Config Connector. In that context, all that matters is that the YAMLs are eventually filled in correctly, either manually or by kpt. AFAIK, kpt won't enforce any state on the YAML until the user runs kpt cfg set, so if they change it manually, it's not an issue from that perspective.

Reply:
SGTM